docs/install.md

```diff
@@ -230,6 +230,24 @@ SHARP_IGNORE_GLOBAL_LIBVIPS=1 npm install --arch=x64 --platform=linux --libc=gli
 To get the best performance select the largest memory available.
 A 1536 MB function provides ~12x more CPU time than a 128 MB function.
 
+### serverless-esbuild
+
+To deploy AWS Lambda using serverless framework, esbuild and serverless-esbuild on machines other than Linux x64 (glibc), first ensure sharp is excluded from bundling via the
```

Comment:
Does `sharp` need to be listed under `externals` on Linux x64 machines too? If so, the wording is a little ambiguous.

Reply:
Need some investigations for this case, if it's required for linux x64 machines as well. So far, in my initial dev deployment, I have configured the following way and it was stable with this configuration.

Reply:
@lovell It does not work if sharp is not put under external with esbuild. Btw, while testing it seems the above setup does not work for v0.30.5.

Currently, I am using v0.30.3 and under the esbuild script tag I have given `- npm install --arch=x64 --platform=linux sharp@0.30.3` to match the version. When I tried to update it with v0.30.5 for both package.json and serverless.yml and run the deployment, the was lambda again started saying something is wrong with the installation `"Cannot find module '../build/Release/sharp-linux-x64.node'"` 

If you have some free time, can you look into this as well. Maybe I am doing something wrong here on my setup. Currently I am using the following versions.

```
dependencies
    "sharp": "^0.30.3",
    
devDependencies
    "serverless": "^3.16.0",
    "serverless-offline": "^8.4.0",
    "esbuild": "^0.14.38",
    "serverless-esbuild": "^1.26.2",
    "@types/sharp": "^0.30.0",
```